src/azure-cli/azure/cli/command_modules/acs/tests/hybrid_2020_09_01/test_aks_commands.py

```diff
@@ -160,6 +160,7 @@ def test_aks_create_default_service_without_skip_role_assignment(self, resource_
             self.check('provisioningState', 'Succeeded')
         ])
 
+    @live_only()
```

Comment:
I have little context of "hybrid_2020_09_01" test, seems it is using fixed version(2020-11-01) of containerservice client, which will cause the test fail because the cli will call `client.list_cluster_user_credentials(
            resource_group_name, name, serverType, credential_format)` but old version client's function only accepts 3 parameters, one example failure: https://dev.azure.com/azure-sdk/public/_build/results?buildId=1385303&view=results. So I marked this test as live_only.

Reply:
Has removed `live_only`. Used `cmd.cli_ctx.cloud.profile` to check whether it is using old sdk to make it can work with old sdk.